routes/net-state-files.go

```diff
@@ -0,0 +1,79 @@
+package routes
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/julienschmidt/httprouter"
+
+	"github.com/storj/storage/boltdb"
+)
+
+var (
+	ErrReadReq = errors.New("error reading request body")
+)
+
+type File struct {
+	DB *boltdb.Client
+}
+
+type Message struct {
+	Value string `json:"value"`
+}
+
+func NewFile(db *boltdb.Client) *File {
+	return &File{DB: db}
+}
+
+func (f *File) Put(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
+	givenPath := ps.ByName("path")
+
+	var msg Message
+	decoder := json.NewDecoder(r.Body)
+	err := decoder.Decode(&msg)
+	if err != nil {
+		log.Printf("err decoding response: %v", err)
+		return
+	}
+
+	file := boltdb.File{
+		Path:  givenPath,
+		Value: msg.Value,
+	}
+
+	if err := f.DB.Put(file); err != nil {
+		fmt.Println(err)
+	}
+
+	fmt.Fprintf(w, "PUT to %s\n", givenPath)
+}
+
+func (f *File) Get(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
+	fileKey := ps.ByName("path")
+
+	fileInfo, err := f.DB.Get([]byte(fileKey))
+	if err != nil {
+		fmt.Println(err)
+	}
+
+	fmt.Fprintf(w, "value: %s", fileInfo)
```

Comment:
could do 

```
bytes, err := json.Marshal(fileInfo)
if err != nil {
  http.Error(w, "internal error", http.StatusInternalServerError)
  return
}
_, err = w.Write(fileInfo)
if err != nil {
  // whoops, we wrote a response already, so can't send a status code now. just log
  log.Printf("failed writing response: %v", err)
}
```

would result in a json response

Reply:
OH okay thank you, I was overthinking this 😅

Reply:
^ would it be `w.Write(bytes)` though instead of `fileInfo` which isn't of type `bytes`?